advanced/is-pattern-2/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@darsham3 shall we please update the license header year to latest? Please update the other files if applicable.

Reply:
updated as 2020 in all relevant files